packages/edit-navigation/src/components/layout/index.js

```diff
@@ -51,6 +59,9 @@ export default function Layout( { blockEditorSettings } ) {
 
 	useMenuNotifications( selectedMenuId );
 
+	const hasMenus = menus?.length;
+	const isBlockEditorReady = !! hasMenus && navigationPost;
```

Comment:
Nit: Should probably wrap it?
```suggestion
	const isBlockEditorReady = !! ( hasMenus && navigationPost );
```

Reply:
👍 I've applied that suggestion.